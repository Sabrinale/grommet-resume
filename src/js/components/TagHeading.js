import React from 'react'

import { Text } from 'grommet'

export const TagHeading = ({ children }) => (
  <Text weight="bold" size="medium">
    {children}
  </Text>
)
