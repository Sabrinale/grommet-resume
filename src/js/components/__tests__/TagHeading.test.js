import React from 'react'
import renderer from 'react-test-renderer'

import { TagHeading } from '../TagHeading'

test('renders tag heading element', () => {
  const component = renderer.create(<TagHeading>test</TagHeading>)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
