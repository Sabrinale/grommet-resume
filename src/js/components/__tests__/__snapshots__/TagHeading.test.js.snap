// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders tag heading element 1`] = `
<span
  className="StyledText-sc-1sadyjn-0 dndifk"
  size="medium"
>
  test
</span>
`;
